### PR TITLE
FIXES #8: Standardize README.md

### PR DESCRIPTION
**Pull request by Code-Hacker26**
Issue #8 addresses the need to standardize and enhance the README.md file. The update includes a structured Table of Contents for easy navigation, clear Installation Instructions with prerequisites, and updated Usage Examples. Badges for forks, stars, issues, and license were added for better project insights. Typographical errors were corrected, and all links were verified. These changes ensure a more professional, user-friendly, and informative README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,303 @@
...
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Here's an overview of what the READMEKRAFT project looks like:
+
+### Home Page
+<img src="public/assets/home.png" alt="Home Page" width="600"/>
+
+### Services
+<img src="public/assets/services.png" alt="Services Page" width="600"/>
```
Comment (TheShiveshNetwork) [inline README.md]: You can't use img tags like this

**Comment by TheShiveshNetwork** [comment]
Hey @Code-Hacker26 just an early comment. I saw that you were confused somewhere so just referring here. You can't render images in markdown like this.
One way is that you use the editor on github to write the readme. Just take a screenshot and drag and drop to the location you want.

**Comment by Code-Hacker26** [comment]
@TheShiveshNetwork I updated the URL link with the Permalink ,plz check if this is feasible solution ?

**Comment by TheShiveshNetwork** [comment]
I don't think it's working. 
image

**Comment by TheShiveshNetwork** [comment]
@Code-Hacker26 It is almost good to go. Can I take it up from here if you don't mind?

**Comment by Code-Hacker26** [comment]
@TheShiveshNetwork  ya sure,

**Review by TheShiveshNetwork — APPROVED** [review]

**Pull request merged**